### PR TITLE
LLM: remove CPU english_quotes dataset and update docker example

### PR DESCRIPTION
**Pull request by hzjane**
## Description
Background: https://github.com/analytics-zoo/nano/issues/1112
Remove `Abirate/english_quotes` dataset in examples and use part of `yahma/alpaca-cleaned` dataset.

### 4. How to test?
- [ ] N/A
- [x] Unit test
- [x] Application test
- [ ] Document test
- [ ] ...

### CODE REVIEW

File: docker/llm/finetune/qlora/xpu/docker/Dockerfile [hunk 1]
```diff
@@ -34,6 +34,9 @@ RUN curl -fsSL https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-P
     pip install git+https://github.com/huggingface/transformers.git@${TRANSFORMERS_COMMIT_ID} && \
     pip install peft==0.5.0 datasets accelerate==0.23.0 && \
     pip install bitsandbytes scipy && \
+    git clone https://github.com/intel-analytics/BigDL.git && \
+    mv BigDL/python/llm/example/GPU/LLM-Finetuning/common /common && \
```
Comment (plusbang) [inline docker/llm/finetune/qlora/xpu/docker/Dockerfile:38]: In `qlora_finetuning_cpu.py`, the `common` path is expected to be in `os.path.join(current_dir, '..', '..', 'GPU', 'LLM-Finetuning')`, I wonder whether works if moving to `/common` here. Other LGTM.

Comment (hzjane) [reply docker/llm/finetune/qlora/xpu/docker/Dockerfile:38]: This is for xpu, it works for the `common_util_path = os.path.join(current_dir, '..', '..').`

Comment (plusbang) [reply docker/llm/finetune/qlora/xpu/docker/Dockerfile:38]: > This is for xpu, it works for the `common_util_path = os.path.join(current_dir, '..', '..').`

Got it, LGTM

**Review by plusbang — APPROVED** [review]

File: docker/llm/finetune/qlora/xpu/docker/Dockerfile (hunk 1 above)
Comment (plusbang) [reply docker/llm/finetune/qlora/xpu/docker/Dockerfile:38]: > This is for xpu, it works for the `common_util_path = os.path.join(current_dir, '..', '..').`

got it, LGTM

**Pull request merged**